Neos.Flow/Classes/I18n/Xliff/XliffParser.php

```diff
@@ -28,67 +26,26 @@
  * @throws Exception\InvalidXliffDataException
  * @see http://docs.oasis-open.org/xliff/v1.2/os/xliff-core.html [1]
  * @see http://docs.oasis-open.org/xliff/v1.2/xliff-profile-po/xliff-profile-po-1.2-cd02.html#s.detailed_mapping.tu [2]
+ * @deprecated use \Neos\Flow\I18n\Xliff\V12\XliffParser instead. Will be removed in Flow 5.0
```

Comment:
The class has no `@api` flag, so in theory we can adjust is as we like. So, yes, this is fine.